### ISSUE TITLE
Locale requirement

### ISSUE THREAD

**Issue by jdeathe** [petems/docker-centos-7-ssh-locale#1]
I'm curious as to why you would require the locale for an image? Am I missing some use cases?

**Comment by petems** [comment]
Yeah, I'm running an app that outputs a debug message when locale is not set: 

```
2017-07-28 23:07:09.182927 WARN  puppetlabs.facter - Could not initialize locale, even with LC_* variables cleared. Continuing without localization support
```

**Comment by jdeathe** [comment]
Thanks! I will give this some thought. I'm not sure it's a requirement of the app if it's just showing a warning and if you're not using a non-default locale in the CLI but I will check this out some more. I think it's possible to add a single locale if you wanted to remove the warning.
